frontend/templates/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="pl">
+  <head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+    <meta name="title" content="Znajdź Praktyki - darmowe narzędzie do szukania praktyk">
+    <meta name="description" content="Przyjazny studentom interfejs z wieloma funkcjami ułatwiającymi znalezienie odpowiedniego praktykodawcy. Projekt w pełni darmowy">
+    <meta name="keywords" content="Praktyki,Internship,Intern">
+    <meta name="robots" content="index, follow">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="language" content="Polish">
+    <meta name="author" content="Blumilk SP. Z.O.O, Bartosz Brzózka, Łukasz Podlipski, Paulina Lencka">
+    <!-- <link rel="icon" href="/favicon.ico" type="image/x-icon" /> -->
+    <link rel="icon" href="../assets/icons/favicon.ico" />
+
+    <link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
+    <!-- <title><%= htmlWebpackPlugin.options.title %></title> -->
+    <title>Znajdź Praktyki</title>
+
+    <link
+      href="https://api.tiles.mapbox.com/mapbox-gl-js/v0.53.0/mapbox-gl.css"
+      rel="stylesheet"
+    />
+    <!-- Vue-mapbox CSS -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/vue-mapbox@latest/dist/vue-mapbox.css"
+      rel="stylesheet"
+    />
+    <!-- Mapbox GL JS -->
+    <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v0.53.0/mapbox-gl.js"></script>
+    <!-- VueJS -->
+    <script src="https://cdn.jsdelivr.net/npm/vue@latest/dist/vue.min.js"></script>
+    <!-- Vue-mapbox -->
+    <script
+      type="text/javascript"
+      src="https://cdn.jsdelivr.net/npm/vue-mapbox@latest/dist/vue-mapbox.min.js"
+    ></script>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.3/css/all.css"
+      integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <noscript>
+      <strong
+        >We're sorry but <%= htmlWebpackPlugin.options.title %> doesn't work
+        properly without JavaScript enabled. Please enable it to
+        continue.</strong
+      >
+    </noscript>
+    <div id="app"></div>
+  </body>
+</html>
```

Comment:
```suggestion
</html>

```

Please, add new line at the end of the file as per general used standard. Github highlights them with red circle with a dash.

You should be able to commit this suggestion directly to avoid losing time with pushes.